### ISSUE TITLE
discern daymet query_service accuracy and improve with url precheck if needed

### ISSUE THREAD

**Issue by ags-tolson** [Applied-GeoSolutions/gips#322]
Presently the daymet driver does minimal checking during the query step, being limited to checking dates per `Asset.start_date` and `Asset.end_date`.  However it may be induced to query the data provider instead, by using pydap to attemp to open a URL and verifying data availability somehow without doing the download itself.  This seems easy for opendap data.

**Comment by bhbraswell** [comment]
Yes that is possible with Pydap. You could also check a subset of data values by spatial index without having to get the whole thing.
